tools/SetupFlow/DevHome.SetupFlow/Controls/SetupShell.xaml

```diff
@@ -76,6 +76,7 @@
             <!-- Allow customizing the header (e.g. add buttons, etc... ) -->

```

Comment:
I'm happy with saying this is a later problem, but if the user did "add buttons, etc" to this ContentControl would this make them not tab stops? Maybe not. Just putting it out there.